src/base.jl

```diff
@@ -314,11 +310,34 @@ Call a function `fn` on `workers`, with a single parameter arriving from the
 corresponding position in `arr`.
 """
 function dmap(arr::Vector, fn, workers)
-    futures = [
-        remotecall(() -> Base.eval(Main, :($fn($(arr[i])))), pid) #TODO convert to get_from
-        for (i, pid) in enumerate(workers)
-    ]
-    return [fetch(f) for f in futures]
+    fetch.([get_from(w, :($fn($(arr[i])))) for (i, w) in enumerate(workers)])
```

Comment:
seems much easier!

Reply:
yeah, certainly it fits in one line now, the way it should be. :D

Still I was wondering if there would be a good way to broadcast over the `enumerate()` result, but this seems most concise.